### ISSUE TITLE
2025-06-09, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#195]
# Today's Arxiv Papers

(1) [Eigenspectrum Analysis of Neural Networks without Aspect Ratio Bias](http://arxiv.org/abs/2506.06280v1), 2025-06-06

(2) [CoMemo: LVLMs Need Image Context with Image Memory](http://arxiv.org/abs/2506.06279v1), 2025-06-06

(3) [Distillation Robustifies Unlearning](http://arxiv.org/abs/2506.06278v1), 2025-06-06

(4) [AdvSumm: Adversarial Training for Bias Mitigation in Text Summarization](http://arxiv.org/abs/2506.06273v1), 2025-06-06

(5) [Cartridges: Lightweight and general-purpose long context representations via self-study](http://arxiv.org/abs/2506.06266v1), 2025-06-06

(6) [PersonaAgent: When Large Language Model Agents Meet Personalization at Test Time](http://arxiv.org/abs/2506.06254v1), 2025-06-06

(7) [DesignBench: A Comprehensive Benchmark for MLLM-based Front-end Code Generation](http://arxiv.org/abs/2506.06251v1), 2025-06-06

(8) [Visual Graph Arena: Evaluating Visual Conceptualization of Vision and Multimodal Large Language Models](http://arxiv.org/abs/2506.06242v1), 2025-06-06

(9) [Bridging External and Parametric Knowledge: Mitigating Hallucination of LLMs with Shared-Private Semantic Synergy in Dual-Stream Knowledge](http://arxiv.org/abs/2506.06240v1), 2025-06-06

(10) [CompilerGPT: Leveraging Large Language Models for Analyzing and Acting on Compiler Optimization Reports](http://arxiv.org/abs/2506.06227v1), 2025-06-06

(11) [PROVSYN: Synthesizing Provenance Graphs for Data Augmentation in Intrusion Detection Systems](http://arxiv.org/abs/2506.06226v1), 2025-06-06

(12) [STSBench: A Spatio-temporal Scenario Benchmark for Multi-modal Large Language Models in Autonomous Driving](http://arxiv.org/abs/2506.06218v1), 2025-06-06

(13) [Can Theoretical Physics Research Benefit from Language Agents?](http://arxiv.org/abs/2506.06214v1), 2025-06-06

(14) [Building Models of Neurological Language](http://arxiv.org/abs/2506.06208v1), 2025-06-06

(15) [Astra: Toward General-Purpose Mobile Robots via Hierarchical Multimodal Learning](http://arxiv.org/abs/2506.06205v1), 2025-06-06

(16) [Does It Run and Is That Enough? Revisiting Text-to-Chart Generation with a Multi-Agent Approach](http://arxiv.org/abs/2506.06175v1), 2025-06-06

(17) [Technical Report for Egocentric Mistake Detection for the HoloAssist Challenge](http://arxiv.org/abs/2506.06174v1), 2025-06-06

(18) [The Lock-in Hypothesis: Stagnation by Algorithm](http://arxiv.org/abs/2506.06166v1), 2025-06-06

(19) [Recommender systems, stigmergy, and the tyranny of popularity](http://arxiv.org/abs/2506.06162v1), 2025-06-06

(20) [Masked Language Models are Good Heterogeneous Graph Generalizers](http://arxiv.org/abs/2506.06157v1), 2025-06-06

(21) [Personalized Large Language Models Can Increase the Belief Accuracy of Social Networks](http://arxiv.org/abs/2506.06153v1), 2025-06-06

(22) [Joint-GCG: Unified Gradient-Based Poisoning Attacks on Retrieval-Augmented Generation Systems](http://arxiv.org/abs/2506.06151v1), 2025-06-06

(23) [Table-r1: Self-supervised and Reinforcement Learning for Program-based Table Reasoning in Small Language Models](http://arxiv.org/abs/2506.06137v1), 2025-06-06

(24) [Let's CONFER: A Dataset for Evaluating Natural Language Inference Models on CONditional InFERence and Presupposition](http://arxiv.org/abs/2506.06133v1), 2025-06-06

(25) [Bridging the Gap: In-Context Learning for Modeling Human Disagreement](http://arxiv.org/abs/2506.06113v1), 2025-06-06

(26) [Towards Lifecycle Unlearning Commitment Management: Measuring Sample-level Unlearning Completeness](http://arxiv.org/abs/2506.06112v1), 2025-06-06

(27) [Text-to-LoRA: Instant Transformer Adaption](http://arxiv.org/abs/2506.06105v1), 2025-06-06

(28) [VideoChat-A1: Thinking with Long Videos by Chain-of-Shot Reasoning](http://arxiv.org/abs/2506.06097v1), 2025-06-06

(29) [Flexible Operator Fusion for Fast Sparse Transformer with Diverse Masking on GPU](http://arxiv.org/abs/2506.06095v1), 2025-06-06

(30) [Reinforcing Code Generation: Improving Text-to-SQL with Execution-Based Learning](http://arxiv.org/abs/2506.06093v1), 2025-06-06

(31) [MIRIAD: Augmenting LLMs with millions of medical query-response pairs](http://arxiv.org/abs/2506.06091v1), 2025-06-06

(32) [Zero-Shot Detection of LLM-Generated Code via Approximated Task Conditioning](http://arxiv.org/abs/2506.06069v1), 2025-06-06

(33) [Conversational Interfaces for Parametric Conceptual Architectural Design: Integrating Mixed Reality with LLM-driven Interaction](http://arxiv.org/abs/2506.06066v1), 2025-06-06

(34) [Simple Yet Effective: Extracting Private Data Across Clients in Federated Fine-Tuning of Large Language Models](http://arxiv.org/abs/2506.06060v1), 2025-06-06

(35) [Hey, That's My Data! Label-Only Dataset Inference in Large Language Models](http://arxiv.org/abs/2506.06057v1), 2025-06-06

(36) [CP-Bench: Evaluating Large Language Models for Constraint Modelling](http://arxiv.org/abs/2506.06052v1), 2025-06-06

(37) [MATP-BENCH: Can MLLM Be a Good Automated Theorem Prover for Multimodal Problems?](http://arxiv.org/abs/2506.06034v1), 2025-06-06

(38) [Large Language Models are Demonstration Pre-Selectors for Themselves](http://arxiv.org/abs/2506.06033v1), 2025-06-06

(39) [When to Trust Context: Self-Reflective Debates for Context Reliability](http://arxiv.org/abs/2506.06020v1), 2025-06-06

(40) [AgentSwift: Efficient LLM Agent Design via Value-guided Hierarchical Search](http://arxiv.org/abs/2506.06017v1), 2025-06-06

(41) [On the Merits of LLM-Based Corpus Enrichment](http://arxiv.org/abs/2506.06015v1), 2025-06-06

(42) [Unlocking Recursive Thinking of LLMs: Alignment via Refinement](http://arxiv.org/abs/2506.06009v1), 2025-06-06

(43) [Token Signature: Predicting Chain-of-Thought Gains with Token Decoding Feature in Large Language Models](http://arxiv.org/abs/2506.06008v1), 2025-06-06

(44) [A Culturally-Rich Romanian NLP Dataset from "Who Wants to Be a Millionaire?" Videos](http://arxiv.org/abs/2506.05991v1), 2025-06-06

(45) [Leveraging Generative AI for Enhancing Automated Assessment in Programming Education Contests](http://arxiv.org/abs/2506.05990v1), 2025-06-06

(46) [Audio-Aware Large Language Models as Judges for Speaking Styles](http://arxiv.org/abs/2506.05984v1), 2025-06-06

(47) [CrimeMind: Simulating Urban Crime with Multi-Modal LLM Agents](http://arxiv.org/abs/2506.05981v1), 2025-06-06

(48) [Mitigating Catastrophic Forgetting with Adaptive Transformer Block Expansion in Federated Fine-Tuning](http://arxiv.org/abs/2506.05977v1), 2025-06-06

(49) [Let's Put Ourselves in Sally's Shoes: Shoes-of-Others Prefixing Improves Theory of Mind in Large Language Models](http://arxiv.org/abs/2506.05970v1), 2025-06-06

(50) [Preference Learning for AI Alignment: a Causal Perspective](http://arxiv.org/abs/2506.05967v1), 2025-06-06

(51) [AQUATIC-Diff: Additive Quantization for Truly Tiny Compressed Diffusion Models](http://arxiv.org/abs/2506.05960v1), 2025-06-06

(52) [Elementary Math Word Problem Generation using Large Language Models](http://arxiv.org/abs/2506.05950v1), 2025-06-06

(53) [IntentionESC: An Intention-Centered Framework for Enhancing Emotional Support in Dialogue Systems](http://arxiv.org/abs/2506.05947v1), 2025-06-06

(54) [Respecting Temporal-Causal Consistency: Entity-Event Knowledge Graphs for Retrieval-Augmented Generation](http://arxiv.org/abs/2506.05939v1), 2025-06-06

(55) [DynamicMind: A Tri-Mode Thinking System for Large Language Models](http://arxiv.org/abs/2506.05936v1), 2025-06-06

(56) [MoA: Heterogeneous Mixture of Adapters for Parameter-Efficient Fine-Tuning of Large Language Models](http://arxiv.org/abs/2506.05928v1), 2025-06-06

(57) [Small Models, Big Support: A Local LLM Framework for Teacher-Centric Content Creation and Assessment using RAG and CAG](http://arxiv.org/abs/2506.05925v1), 2025-06-06

(58) [Generating Grounded Responses to Counter Misinformation via Learning Efficient Fine-Grained Critiques](http://arxiv.org/abs/2506.05924v1), 2025-06-06

(59) [Route-and-Reason: Scaling Large Language Model Reasoning with Reinforced Model Router](http://arxiv.org/abs/2506.05901v1), 2025-06-06

(60) [Object Navigation with Structure-Semantic Reasoning-Based Multi-level Map and Multimodal Decision-Making LLM](http://arxiv.org/abs/2506.05896v1), 2025-06-06

(61) [Explainability in Context: A Multilevel Framework Aligning AI Explanations with Stakeholder with LLMs](http://arxiv.org/abs/2506.05887v1), 2025-06-06

(62) [Human-AI Alignment of Multimodal Large Language Models with Speech-Language Pathologists in Parent-Child Interactions](http://arxiv.org/abs/2506.05879v1), 2025-06-06

(63) [Bayesian Persuasion as a Bargaining Game](http://arxiv.org/abs/2506.05876v1), 2025-06-06

(64) [Research on Personalized Financial Product Recommendation by Integrating Large Language Models and Graph Neural Networks](http://arxiv.org/abs/2506.05873v1), 2025-06-06

(65) [BestServe: Serving Strategies with Optimal Goodput in Collocation and Disaggregation Architectures](http://arxiv.org/abs/2506.05871v1), 2025-06-06

(66) [Towards Next-Generation Intelligent Maintenance: Collaborative Fusion of Large and Small Models](http://arxiv.org/abs/2506.05854v1), 2025-06-06

(67) [Training-Free Query Optimization via LLM-Based Plan Similarity](http://arxiv.org/abs/2506.05853v1), 2025-06-06

(68) [Cross-lingual Collapse: How Language-Centric Foundation Models Shape Reasoning in Large Language Models](http://arxiv.org/abs/2506.05850v1), 2025-06-06

(69) [Heartcare Suite: Multi-dimensional Understanding of ECG with Raw Multi-lead Signal Modeling](http://arxiv.org/abs/2506.05831v1), 2025-06-06

(70) [CodeContests+: High-Quality Test Case Generation for Competitive Programming](http://arxiv.org/abs/2506.05817v1), 2025-06-06

(71) [MAPLE: Multi-Agent Adaptive Planning with Long-Term Memory for Table Reasoning](http://arxiv.org/abs/2506.05813v1), 2025-06-06

(72) [Diarization-Aware Multi-Speaker Automatic Speech Recognition via Large Language Models](http://arxiv.org/abs/2506.05796v1), 2025-06-06

(73) [Discrete Minds in a Continuous World: Do Language Models Know Time Passes?](http://arxiv.org/abs/2506.05790v1), 2025-06-06

(74) [EASG-Bench: Video Q&A Benchmark with Egocentric Action Scene Graphs](http://arxiv.org/abs/2506.05787v1), 2025-06-06

(75) [dots.llm1 Technical Report](http://arxiv.org/abs/2506.05767v1), 2025-06-06

(76) [BioMol-MQA: A Multi-Modal Question Answering Dataset For LLM Reasoning Over Bio-Molecular Interactions](http://arxiv.org/abs/2506.05766v1), 2025-06-06

(77) [Writing-RL: Advancing Long-form Writing via Adaptive Curriculum Reinforcement Learning](http://arxiv.org/abs/2506.05760v1), 2025-06-06

(78) [Efficient Online RFT with Plug-and-Play LLM Judges: Unlocking State-of-the-Art Performance](http://arxiv.org/abs/2506.05748v1), 2025-06-06

(79) [LLM-Symbolic Integration for Robust Temporal Tabular Reasoning](http://arxiv.org/abs/2506.05746v1), 2025-06-06

(80) [To Protect the LLM Agent Against the Prompt Injection Attack with Polymorphic Prompt](http://arxiv.org/abs/2506.05739v1), 2025-06-06

(81) [Do LLMs Really Forget? Evaluating Unlearning with Knowledge Correlation and Confidence Awareness](http://arxiv.org/abs/2506.05735v1), 2025-06-06

(82) [Large Language Models are Good Relational Learners](http://arxiv.org/abs/2506.05725v1), 2025-06-06

(83) [Bridging the Modality Gap: Softly Discretizing Audio Representation for LLM-based Automatic Speech Recognition](http://arxiv.org/abs/2506.05706v1), 2025-06-06

(84) [RKEFino1: A Regulation Knowledge-Enhanced Large Language Model](http://arxiv.org/abs/2506.05700v1), 2025-06-06

(85) [Being Strong Progressively! Enhancing Knowledge Distillation of Large Language Models through a Curriculum Learning Framework](http://arxiv.org/abs/2506.05695v1), 2025-06-06

(86) [SafeGenBench: A Benchmark Framework for Security Vulnerability Detection in LLM-Generated Code](http://arxiv.org/abs/2506.05692v1), 2025-06-06

(87) [When to use Graphs in RAG: A Comprehensive Analysis for Graph Retrieval-Augmented Generation](http://arxiv.org/abs/2506.05690v1), 2025-06-06

(88) [Pts3D-LLM: Studying the Impact of Token Structure for 3D Scene Understanding With Large Language Models](http://arxiv.org/abs/2506.05689v1), 2025-06-06

(89) [Voice Impression Control in Zero-Shot TTS](http://arxiv.org/abs/2506.05688v1), 2025-06-06

(90) [Zero-Shot Event Causality Identification via Multi-source Evidence Fuzzy Aggregation with Large Language Models](http://arxiv.org/abs/2506.05675v1), 2025-06-06

(91) [Contextually Guided Transformers via Low-Rank Adaptation](http://arxiv.org/abs/2506.05672v1), 2025-06-06

(92) [Low-Resource Domain Adaptation for Speech LLMs via Text-Only Fine-Tuning](http://arxiv.org/abs/2506.05671v1), 2025-06-06

(93) [Can LLMs Express Personality Across Cultures? Introducing CulturalPersonas for Evaluating Trait Alignment](http://arxiv.org/abs/2506.05670v1), 2025-06-06

(94) [BAQ: Efficient Bit Allocation Quantization for Large Language Models](http://arxiv.org/abs/2506.05664v1), 2025-06-06

(95) [Hallucinate, Ground, Repeat: A Framework for Generalized Visual Relationship Detection](http://arxiv.org/abs/2506.05651v1), 2025-06-06

(96) [FedShield-LLM: A Secure and Scalable Federated Fine-Tuned Large Language Model](http://arxiv.org/abs/2506.05640v1), 2025-06-06

(97) [Joint User Association and Beamforming Design for ISAC Networks with Large Language Models](http://arxiv.org/abs/2506.05637v1), 2025-06-05

(98) [IYKYK: Using language models to decode extremist cryptolects](http://arxiv.org/abs/2506.05635v1), 2025-06-05

(99) [Leveraging Self-Attention for Input-Dependent Soft Prompting in LLMs](http://arxiv.org/abs/2506.05629v1), 2025-06-05

(100) [Deployability-Centric Infrastructure-as-Code Generation: An LLM-based Iterative Framework](http://arxiv.org/abs/2506.05623v1), 2025-06-05
